### ISSUE TITLE
SilverBlue 38 [noob question :)]

### ISSUE THREAD

**Issue by 0xsimulacra** [CheariX/silverblue-akmods-keys#9]
Hello,

I recently (2 days ago) switched to SilverBlue and I'm a total noob with regards to immutable distros, my longest experience on linux was using RHEL 8, I discovred SilverBlue when I was experimenting with CoresOS (It was a total faillure for me as I didn't manage to the stuff I wanted to do on my own and the documentation is rare). Anyway, I loved the concept, the rpm-ostree and the container oriented side of the OS and since I like RHEL too, it was an easy dicision to switch to SilverBlue.

I wish if there was a book about SilverBlue that would help me master it. Altough, I feel like mastering SilverBlue is just a matter of having a good theoritical knowldge of the Linux system and not only pratical knowldge.

My question about the repos and the packages provided is will I be  required to redo the steps detailed in the repos if I upgrade one day to the hopefully soon to be release Fedora SilverBlue 38 ? Since we created an RPM for the 37 fedora version, I imagine we will need to create another one for the 38 version, or maybe we will need to uninstall the one we created before upgrading then create a new one after the upgrade ?

Thank you very much for the repos <3

**Comment by CheariX** [comment]
Hi,

I feel with you - the beginning with Silverblue is not so easy. But it's worth it in the end.

Regarding your question about `silverblue-akmods-keys`:
This package is very tiny and does not do very much, so it's not very likely to break with future version of Silverblue as long as upstream `akmods` does not change the way it handles keys.
For me, I created and installed this package once with F36, upgraded to F37, upgraded to F38. I did not experience a single issue with this package. You do not need to reinstall/rebuild/anything during these upgrades.
